community/neo4j/src/test/java/org/neo4j/index/LegacyIndexTest.java

```diff
@@ -38,12 +38,10 @@
 
 public class LegacyIndexTest
 {
-    private static final long TEST_TIMEOUT = 40_000;
-
     @Rule
     public TestDirectory directory = TestDirectory.testDirectory();
 
-    @Test( timeout = TEST_TIMEOUT )
+    @Test
```

Comment:
test timeout was there to verify that performance of legacy index population is more or less decent for cases when we have a lot of fields in the index. With proposed change timeout is gone making this test totally useless. Not sure its a good thing